apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -329,7 +330,7 @@ upload_install(post, #{body := #{<<"plugin">> := Plugin}}) when is_map(Plugin) -
             case emqx_plugins:parse_name_vsn(FileName) of
                 {ok, AppName, _Vsn} ->
                     AppDir = filename:join(emqx_plugins:install_dir(), AppName),
-                    case filelib:wildcard(AppDir ++ "*.tar.gz") of
```

Comment:
If plugin `plugin_a-5.0.0` has been already installed and `plugin-5.0.0` is being installed, wildcard `plugins/plugin*.tar` would match `plugins/plugin_a-5.0.0.tar.gz` and, thus, block installing `plugin-5.0.0.tar.gz`.
The suggested wildcard `-*[0-9]*.[0-9]*.[0-9]*.tar.gz` should match any valid 'semver like' version: `v5.0.0`, `5.0.0-beta`, `v5.0.0-beta` etc.